include/Core/UI/Page/LogListPage.php

```diff
@@ -103,6 +103,7 @@ public function render_page() {
 			</form>
 		</div>
 		<?php
+		parent::render_page_footer();
```

Comment:
Oops, oh yeah, `render_page_footer()` was not overridden in the Child class. I'll update it and send another PULL request.